contracts/adapters/FixedRebalanceExtension.sol

```diff
@@ -115,16 +115,15 @@ contract FixedRebalanceExtension is BaseExtension {
     }
 
     /**
-     * ONLY OPERATOR: Rebalances the positions towards the configured allocation percentages.
+     * ONLY EOA AND ALLOWED CALLER: Rebalances the positions towards the configured allocation percentages.
      *
      * @param _share                Relative share of the necessary trade volume to execute (allows for splitting the rebalance over multiple transactions
      * @param _minPositions         Minimum positions (in set token units) for each maturity after rebalance. (slippage protection)
      */
-    function rebalance(uint256 _share, uint256[] memory _minPositions) external onlyOperator {
+    function rebalance(uint256 _share, uint256[] memory _minPositions) external onlyAllowedCaller(msg.sender) {
```

Comment:
You'll need to add onlyEOA to only allow EOA - although should this only be restricted to EOA?

Reply:
Ah yeah no. I removed the onlyEOA modifier since I assume we might want to call this from a multisig wallet. Forget to adjust the docstring though. Will do that.